rendermodules/utilities/pillow_utils.py

```diff
@@ -1,3 +1,14 @@
 from PIL import Image
 
 Image.MAX_IMAGE_PIXELS = None
+
+# imports needed for rough_align_qc
```

Comment:
These matplotlib imports should not be necessary now.

Reply:
done.